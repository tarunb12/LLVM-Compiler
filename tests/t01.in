// Syntax error test

int main() {
    // hello world @@
    @@ look at this error
    printf("test")
}