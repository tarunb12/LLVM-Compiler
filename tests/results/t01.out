Syntax error: File "tests/t01.in", line 4: "@".